2-api/methods.md

```diff
@@ -2,7 +2,9 @@
 
 # API Methods
 
-There are two API methods: 
+A method is a procedure associating a message (or request) with an object (which consists of data and behavior). Once run, a method returns a new object.
```

Comment:
This feels too abstract to me. What if instead we have a high level description of what the two different options for requests are? Something like the following (and feel free to machete this):

"There are two methods available for Perspective API, where each method is a different type of request a user can send:
- AnalyzeComment, where a user sends a request for a comment to be analyzed, and a score is returned
- SuggestCommentScore, where a user can suggest a better score fora  comment"